hazelcast-client-legacy/src/test/java/com/hazelcast/client/multimap/ClientMultiMapLockTest.java

```diff
@@ -324,8 +324,13 @@ public void testLockTTLExpired_whenLockedBySelf() throws Exception {
 
         mm.lock(key);
         mm.lock(key, 1, TimeUnit.SECONDS);
-        sleepSeconds(2);
-        assertFalse(mm.isLocked(key));
+
+        assertTrueEventually(new AssertTask() {
+            @Override
+            public void run() throws Exception {
+                assertFalse(mm.isLocked(key));
+            }
+        });
     }
```

Comment:
the previous test testLockTTLExpired() can have the same problem, is it possible to change that too?